File: 9. Palindrome Number/main.js
/**
 * @param {number} x
 * @return {boolean}
 */
var isPalindrome = function (x) {
  let sX = x.toString();
  let i = 0,
    j = sX.length - 1;

  while (i < j) if (sX[i++] != sX[j--]) return false;

  return true;
};

File: 9. Palindrome Number/main.cpp
#include <string>
using namespace std;

class Solution {
public:
    bool isPalindrome(int x) {
        string xS = to_string(x);
        int i = 0, j = xS.length() - 1;

        while(i < j) if(xS[i++] != xS[j--]) return false;

        return true;
    }
};